### DISCUSSION TITLE
Can't regrid dataset

### DISCUSSION THREAD

**Discussion by jbadag** [pangeo-data/xESMF#312] [Q&A]
Hi, 
I am trying to regrid a netcdf file downloaded from https://smn-ar-wrf.s3-us-west-2.amazonaws.com/index.html#DATA/WRF/DET/2023/10/28/00/. Specifically, I am attempting to regrid the file WRFDETAR_01H_20231028_00_001.nc to a rectangular coordinate system in a region bounded by (-60º, -48º) in longitude and (-36, -25º) in latitude. To do this, I wrote the following Python code:

```python3
import xarray as xr
import numpy as np
import xesmf as xe

ds = xr.open_dataset('WRFDETAR_01H_20231028_00_001.nc', decode_coords = 'all', engine = 'netcdf4')

lon_new = np.arange(-60, -47.9, 0.1)
lat_new = np.arange(-36, -24.9, 0.1)

regridder = xe.Regridder(ds, {'lon': lon_new, 'lat': lat_new}, method='bilinear')
ds_out = regridder(ds)
```

However, when I run it, I get an error message (attached a screenshot). Does anyone know why this is happening?
<img width="578" alt="Captura de pantalla 2023-10-31 114851" src="https://github.com/pangeo-data/xESMF/assets/16575671/40ed35f0-124b-463a-8765-0ec9fa4c8f29">

**Comment by aulemahal** [comment]
Hi @jbadag, I wasn't able to reproduce your bug, this code works on my side...

Would you be able to update xESMF and try again ?

For reference, I am using xESMF 0.8.2. And my `xr.show_versions()` gives:

<details>

INSTALLED VERSIONS
------------------
commit: None
python: 3.11.6 | packaged by conda-forge | (main, Oct  3 2023, 10:40:35) [GCC 12.3.0]
python-bits: 64
OS: Linux
OS-release: 6.5.4-200.fc38.x86_64
machine: x86_64
processor: 
byteorder: little
LC_ALL: None
LANG: fr_CA.UTF-8
LOCALE: ('fr_CA', 'UTF-8')
libhdf5: 1.14.2
libnetcdf: 4.9.2

xarray: 2023.8.0
pandas: 2.1.1
numpy: 1.24.4
scipy: 1.11.3
netCDF4: 1.6.5
pydap: None
h5netcdf: 1.2.0
h5py: 3.10.0
Nio: None
zarr: 2.16.1
cftime: 1.6.2
nc_time_axis: 1.4.1
PseudoNetCDF: None
iris: None
bottleneck: 1.3.7
dask: 2023.10.0
distributed: 2023.10.0
matplotlib: 3.8.0
cartopy: 0.22.0
seaborn: None
numbagg: None
fsspec: 2022.11.0
cupy: None
pint: 0.22
sparse: 0.14.0
flox: 0.8.1
numpy_groupies: 0.10.2
setuptools: 68.2.2
pip: 23.3.1
conda: installed
pytest: 7.4.3
mypy: None
IPython: 8.16.1
sphinx: 5.3.0

</details>
